redi-connect-front/src/components/organismms/LoggedOutWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import LoggedOutNavbar from "./LoggedOutNavbar";
+import Footer from "./Footer";
+
+const LoggedOutWrapper = ({ children }: any) => {
```

Comment:
Look [here](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#function-components) for a better way to declare this component and getting `children` into a variable. We should use `any` as little as possible.

Reply:
Changed. Replaced with ```export const PageViewTracker: React.FC = ({ children }) => { ...}```